fe/fe-core/src/main/java/com/starrocks/statistic/StatisticSQLBuilder.java

```diff
@@ -27,18 +27,18 @@ public class StatisticSQLBuilder {
 
     private static final String QUERY_FULL_STATISTIC_TEMPLATE =
             "SELECT cast(" + STATISTIC_DATA_VERSION + " as INT), $updateTime, db_id, table_id, column_name,"
-                    + " sum(row_count), cast(avg(data_size) as bigint), hll_union_agg(ndv), sum(null_count), "
+                    + " sum(row_count), cast(sum(data_size) as bigint), hll_union_agg(ndv), sum(null_count), "
```

Comment:
why change avg to sum?

Reply:
This is a bug, I misunderstood the meaning of data_size in the statistics of the old version. What it really means: "sum of size of all rows", what I understand means "average size of this column across all rows". In order to ensure consistency with the statistics estimation logic of the old version, it should be sum instead of avg